js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -992,10 +992,30 @@ function processQuery (query, explain) {
     }
     return '';
   };
+  
+  var iterateIndices = function (idx, i, node, types, variable) {
+    var what = (node.reverse ? 'reverse ' : '') + idx.type + ' index scan' + ((node.producesResult || !node.hasOwnProperty('producesResult')) ? (node.indexCoversProjections ? ', index only' : '') : ', scan only');
+    if (types.length === 0 || what !== types[types.length - 1]) {
+      types.push(what);
+    }
+    idx.collection = node.collection;
+    idx.node = node.id;
+    if (node.hasOwnProperty('condition') && node.condition.type && node.condition.type === 'n-ary or') {
+      idx.condition = buildExpression(node.condition.subNodes[i]);
+    } else {
+      if (variable !== false) {
```

Comment:
Why `false` here?
If the function is called with just 4 parameters, `variable` will be `undefined`, not `false`

Reply:
yes, false is missing here.